### DISCUSSION TITLE
Using shared presets on AWS CodeCommit

### DISCUSSION THREAD

**Discussion by Marvin0098** [renovatebot/renovate#23098] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

AWS CodeCommit

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hello,

I want to use shared presets on AWS CodeCommit. Is this even supported on the platform? I have not found anything in this regards in the documentation.

When I try to extend a renovate.json file in one of my repos with the following syntax `"extends": ["codecommit>renovate-config"]`. 
I get this error: `Cannot find preset's package (l)`

When I try with local preset: `"extends": ["local>renovate-config"]`. 
I get this error `"The platform you're using ($platform) does not support local presets"`

Has anyone done this on AWS codecommit and can maybe help me out with some snippets/tips?

Thank you

### What you tried so far.

Using both `codecommit `and `local `preset

**Comment by rarkins** [comment]
It's not supported: https://github.com/renovatebot/renovate/blob/3c3f2464481bc2ab0f1cf9c292478ea99a56b695/lib/config/presets/local/index.ts#L19-L28

@PhilipAbed do you remember if it's simply not supported yet (in which case we should add a feature request issue to track) or whether it's _impossible_ (in which case we just document)

**Reply by Marvin0098** [reply]
Thanks for the quick answer.

**Reply by PhilipAbed** [reply]
@rarkins 
i wasn't aware of this functionality requirements, so it has not been tested or implemented.

**Comment by rarkins** [comment, marked as answer]
Feature request created: #23099
